core/blockchain_sethead_test.go

```diff
@@ -248,280 +220,11 @@ func TestShortFastSyncingSetHead(t *testing.T) {
 	// Expected head block     : G
 	testSetHead(t, &rewindTest{
 		canonicalBlocks:    8,
-		sidechainBlocks:    0,
-		freezeThreshold:    16,
```

Comment:
Ditto, why remove anything other than `sidechainBlocks` and `expSideChainBlocks`?

Reply:
That's git showing some deletes in a weird manner.
Basically, we had tests with only the canonical chain, and others that were the same, but adding the sidechain.
I removed those with the sidechain, and github instead of remove the entire test, cuts the first part of this one, and paste the last part of the other. But basically is removing the unused sidechain parameters, and then github collapsing things in a strange way